app/src/main/java/com/kickstarter/libs/ApiPaginator.java

```diff
@@ -34,6 +34,7 @@
   private final @NonNull Func1<List<Data>, List<Data>> pageTransformation;
   private final boolean clearWhenStartingOver;
   private final @NonNull Func2<List<Data>, List<Data>, List<Data>> concater;
+  private final boolean distinctUntilChanged;
```

Comment:
@mbrandonw lmk if i am adding this (Android paginator version of `skipRepeats`) correctly here